### PR TITLE
feat(size-and-space): apply between stops and negative variables vue3

### PR DESCRIPTION
**Pull request by francisrupert**
# PR Title

For [DT-1142](https://dialpad.atlassian.net/browse/DT-1142), applying the `dialtone`'s update from [DT-1131](https://dialpad.atlassian.net/browse/DT-1131) and [DT-1143](https://dialpad.atlassian.net/browse/DT-1143).

## :hammer_and_wrench: Type Of Change

- [ ] Fix
- [ ] Feature
- [x] Refactoring
- [ ] Documentation

## :pencil: Checklist

- [ ] I have reviewed my changes
- [ ] I have added tests
- [ ] I have added all relevant documentation
- [ ] I have validated components with a screen reader
- [ ] I have validated components keyboard navigation
- [ ] I have considered the performance impact of my change
- [ ] I have checked that my change did not significantly increase bundle size
- [ ] I am exporting any new components or constants in the index.js in the component directory
- [ ] I am exporting any new components or constants in the index.js in the root

## :link: Sources

[DT-1142]: https://dialpad.atlassian.net/browse/DT-1142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DT-1131]: https://dialpad.atlassian.net/browse/DT-1131?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DT-1143]: https://dialpad.atlassian.net/browse/DT-1143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/welcome.mdx [hunk 1]
```diff
@@ -27,12 +27,20 @@ Dialpad projects.
...
 - **Ignacio Ropolo** - Engineering
 
+### Product Design
+
+- **Francis Rupert** - Manager, Design Systems and Platform
+- **Marshall Norman** - Product Design
+- **Bradley Hawkins** - Product Design
+- **Federico Sarassola** - Product Design
+
 [//]: # (TODO: Replace this with the overview link once is created on dialtone.)
```
Comment (juliodialpad) [inline docs/welcome.mdx]: Same as Vue2 branch, please remove comment

**Review by juliodialpad — CHANGES_REQUESTED** [review]
Hmm, seems like Vue2 changes weren't cherry-picked correctly. Only 3 files modified

**Comment by francisrupert** [comment]
> Hmm, seems like Vue2 changes weren't cherry-picked correctly. Only 3 files modified

Resumed cherry-pick, it's been waiting for me to continue for a few days now :P

**Review by braddialpad — APPROVED** [review]
Looks good now thanks!

**Review by braddialpad — APPROVED** [review]
Looks good now thanks!

**Pull request merged**